src/GregClient/GregClient.cs

```diff
@@ -40,7 +40,15 @@ private IRestResponse ExecuteInternal(Request m)
 
```

Comment:
cheap and dirty ?

not yet fully done...

Reply:
When we refactor the `SignRequest` method, we will be able to pass down the `authParams` (or something similar) as an optional arg...